languages/elixir/exercises/concept/booleans/.meta/design.md

```diff
@@ -2,13 +2,13 @@
 
 ## Goal
 
-The goal of this exercise is to teach the student the basics of booleans and logicical expressions in Elixir through the expression of game rule logic(1).
+The goal of this exercise is to teach the student the basics of booleans and boolean logical expressions in Elixir through the expression of game rule logic(1).
 
 ## Learning objectives
 
 - Know what a variable is.
 - Know how to define a named function.
-- Know how to define a function with parameter variables.
+- Know how to define a function with parameters.
 - Know how to return a value from a function.
 - Know how to define a boolean.
```

Comment:
I'm not sure exactly what this means.

Reply:
Probably just a copy and paste remnant. Thanks for catching it.